Assets/UnityIO.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEngine;
+using System;
+using System.IO;
+using System.Collections;
+
+public class UnityIO
+{
+	public class Response {
+		public string error;
+		public byte[] data;
+
+		public Response(string e, byte[] d) {
+			error = e;
+			data = d;
+		}
+		public Response(string e, string d) {
+			error = e;
+			data = Convert.FromBase64String(d);
+		}
+	}
+
+	public delegate void IORequestCallback(object userData, Response response);
+
+	public IEnumerator FetchData(Uri uri, IORequestCallback callback, object userData) {
+
+		bool networkRequest = false;
+		Response response;
+		if (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps) {
+			networkRequest = true;
+		}
+
+		if (networkRequest) {
+			WWW request = new WWW(uri.AbsoluteUri);
```

Comment:
In practice and in documentation I did not find any difference. Documentation mentions the use of both. `WWW` had just a bit less boiler plate code compared to `UnityWebRequest` hence I chose to use `WWW` over `UnityWebRequest`.